trollflow2/launcher.py

```diff
@@ -37,23 +37,25 @@
 from collections import OrderedDict
 from contextlib import suppress
 from datetime import datetime
-from logging import getLogger
+import logging
 from queue import Empty
+from multiprocessing import Manager
 
 import yaml
-from trollflow2.dict_tools import gen_dict_extract, plist_iter
-from trollflow2.logging import setup_queued_logging
-from trollflow2.plugins import AbortProcessing
+from yaml import UnsafeLoader, SafeLoader, BaseLoader
 
 try:
     from posttroll.listener import ListenerContainer
 except ImportError:
     ListenerContainer = None
 
-from yaml import UnsafeLoader, SafeLoader, BaseLoader
+from trollflow2.dict_tools import gen_dict_extract, plist_iter
+from trollflow2.logging import logging_on
+from trollflow2.logging import setup_queued_logging
+from trollflow2.plugins import AbortProcessing
 
 
-LOG = getLogger(__name__)
+LOG = logging.getLogger(__name__)
```

Comment:
From a coding style point of view, this should probably be `logger` rather than `LOG` (it is not a constant).

Reply:
We use `LOG` throughout the package. I'll create a new issue out of this.

Reply:
https://github.com/pytroll/trollflow2/issues/163